### PR TITLE
Fix Image range selection

### PR DESCRIPTION
**Pull request by juliaroldi**
Fix 1: when select image for copy/cut, the image range selection must appear. 
Fix 2: on firefox delete the image using delete/backspace to remove the image.

SelectImageWithCopyCut

Warning: this fix for image range selection does not work on Safari, selecting image range on safari is breaking the cut behavior.

### CODE REVIEW

File: packages/roosterjs-content-model-core/lib/corePlugin/selection/SelectionPlugin.ts [hunk 1]
```diff
@@ -768,21 +783,6 @@ class SelectionPlugin implements PluginWithState<SelectionPluginState> {
             this.state.mouseDisposer = undefined;
         }
     }
-
-    private trySelectSingleImage(selection: RangeSelection) {
```
Comment (BryanValverdeU) [inline packages/roosterjs-content-model-core/lib/corePlugin/selection/SelectionPlugin.ts:772]: When selecting the image with Ctrl + Arrow Keys now we do not show image selection and instead we show range selection. Is this expected? 

I believe it is different than Word, in Word as soon as the image is selected with either Shift + Arrow, Word adds the wrapper for resizing the image.

**Comment by JiuqingSong** [comment]
About the Safari warning, does it regress anything, or just not a complete fix?

**Comment by juliaroldi** [comment]
> About the Safari warning, does it regress anything, or just not a complete fix?

The fix will not work for Safari, if the image is selected with range when cut/copy, the cut will not work, so for Safari, we do not change anything, just leave the behavior we have now.

File: packages/roosterjs-content-model-core/lib/corePlugin/selection/SelectionPlugin.ts [hunk 2]
```diff
@@ -362,6 +361,23 @@ class SelectionPlugin implements PluginWithState<SelectionPluginState> {
                         this.selectBeforeOrAfterElement(editor, selection.image);
                     }
                 }
+
+                if (
+                    (isModifierKey(rawEvent) || rawEvent.shiftKey) &&
+                    selection.image &&
+                    !this.isSafari
```
Comment (juliaroldi) [inline packages/roosterjs-content-model-core/lib/corePlugin/selection/SelectionPlugin.ts:368]: @JiuqingSong Safari will ignore range selection

**Review by JiuqingSong — APPROVED** [review]

**Pull request merged**